docs/content/start/telemetry.md

```diff
@@ -0,0 +1,255 @@
+
+
+```ts
+export interface TinyliciousContainerConfig {
+  id: string;
+  logger?: ITelemetryBaseLogger;
+}
+```
+`TinyliciousContainerConfig` interface definition takes an optional parameter `logger`. (The definition is similar to `FrsContainerConfig` interface)
+
+
+```ts
+public async createContainer(
+  serviceContainerConfig: TinyliciousContainerConfig,
+  containerSchema: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+`createContainer()` interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The `FrsClient` will take `FrsContainerConfig` for its respective methods)
+
+```ts
+static async getContainer(
+  serviceConfig: TinyliciousContainerConfig,
+  objectConfig: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+`getContainer()` interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The `FrsClient` will take `FrsContainerConfig` for its respective methods)
+
+
+```ts
+const loader = new Loader({
+  urlResolver: this.urlResolver,
+  documentServiceFactory: this.documentServiceFactory,
+  codeLoader,
+  logger: tinyliciousContainerConfig.logger,
+});
+```
+The `Loader` object is called by both `createContainer()` and `getContainer()`, and requires a `ILoaderProps` interface as its constructor argument. `ILoaderProps` interface has an optional logger parameter that will take the `ITelemetryBaseLogger` defined by the user.
+
+```ts
+constructor(loaderProps: ILoaderProps) {
+  const scope = { ...loaderProps.scope };
+  if (loaderProps.options?.provideScopeLoader !== false) {
+      scope.ILoader = this;
+  }
+
+  this.services = {
+      urlResolver: createCachedResolver(MultiUrlResolver.create(loaderProps.urlResolver)),
+      documentServiceFactory: MultiDocumentServiceFactory.create(loaderProps.documentServiceFactory),
+      codeLoader: loaderProps.codeLoader,
+      options: loaderProps.options ?? {},
+      scope,
+      subLogger: DebugLogger.mixinDebugLogger("fluid:telemetry", loaderProps.logger, { all:{loaderId: uuid()} }),
+      proxyLoaderFactories: loaderProps.proxyLoaderFactories ?? new Map<string, IProxyLoaderFactory>(),
+      detachedBlobStorage: loaderProps.detachedBlobStorage,
+  };
+  this.logger = ChildLogger.create(this.services.subLogger, "Loader");
+}
+```
+`ILoaderProps.logger` is used by `Loader` to pipe to container's telemetry system.
+
+
+
+### Properties and methods
+
+The interface contains a `supportTags` property and a `send()` method as shown:
+
+```ts
+export interface ITelemetryBaseLogger {
+  supportsTags?: true;
```

Comment:
It's now officially deprecated in main (still needs to be backported to previous release, but happening soon):  https://github.com/microsoft/FluidFramework/blob/24f5cd9acd3a7ef83604edd9863a47ea50c30569/common/lib/common-definitions/src/logger.ts#L43-46

Reply:
sure, I'll remove it.